js/store.js

```diff
@@ -292,29 +291,23 @@ const store = {
       thirdParty['firstPartyHostnames'].push(origin);
     }
 
-    // add link in first party if it doesn't exist yet
-    // and the third party is visible (i.e. not allowlisted)
-    if (!this.isFirstPartyLinkedToThirdParty(firstParty, target)) {
-      if (!this.isVisibleThirdParty(thirdParty)) {
```

Comment:
I preferred this method for readability, removing it might change the logic what is the rationale? Is it still used?

Reply:
This previously was in place to check for missing `isVisible` keys. Perhaps this is no longer needed though.